### DISCUSSION TITLE
@include * ignored by some sites?

### DISCUSSION THREAD

**Discussion by 3dom3k** [violentmonkey/violentmonkey#1464] [Q&A] [answered]
Hi,

I run violentmonkey in FF83 32bit (from MX Linux 19.3) and it basically works flawlessly. Except one strange thing..

I have one userscript which I want to run on every page. It works well. It's metadata section looks like:

```
// ==UserScript==
// @id              all-styles
// @name            all-styles
// @version         1.0
// @namespace       all
// @author          3dom3k
// @description     General steps for every page (dark skin, font, spacing, disable crazy shortcuts, etc).
// @include         *
// @run-at          document-end
// ==/UserScript==
```
Now the strange thing: The `@include *` metatag should ensure that a userscript is fired on every web page, right?
Actually, it almost does (and a number over the monkey icon indicates the number of fired userscripts), except that it does not fire on site `https://support.mozilla.org/gu-IN/questions/1320335` and generally, on any site starting with url `https://support.mozilla.org/` (There is no number over monkey icon and the effect of userscript is not seen).

How it is possible? I have downloaded the sources (violentmonkey-master.zip) and unzippped them, and grepped for occurences of "mozilla" (case insensitively), but could not found anything indicating, that urls containing `support.mozilla.org` would have any kind of exemption from `@include *` metatag.

Any ideas, why this happens to me? It's not a disaster, but I am just curious..

**Comment by tophf** [comment, marked as answer]
In Firefox 60 and newer you need to remove this domain from `extensions.webextensions.restrictedDomains` in `about:config`

**Comment by 3dom3k** [comment]
Oh!

tophf, thank you for quick and exact answer! I am impressed! Thank you again.
